src/create/mod.rs

```diff
@@ -40,4 +38,4 @@ impl Drop for RemoveOnDrop<'_> {
 
 const CONFIG: &str = include_str!("Yarner.toml");
 
-const DOCUMENT: &str = include_str!("document.md");
+const DOCUMENT: &str = include_str!("README.md");
```

Comment:
Since we are not replacing anything anymore, we can use `include_bytes!` here and drop the `.as_bytes()` call above.